### DISCUSSION TITLE
Can I make a PR with edits to the front page of this repo?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#270] [Help]
I made edits to the front page but it did not let me make a PR with my changes. Is there a way that I can?

**Comment by apple502j** [comment]
What's front page?

**Comment by Explosion-Scratch** [comment]
like the code tab. Just the normal view of the repo, not the issues tab or anything.

**Reply by cobaltt7** [reply]
i know this is old but just FYI, that's called the README

**Reply by GrahamSH-LLK** [reply]
Please please please don't necropost

**Comment by WorldLanguages** [comment]
Hmm what are you trying to edit exactly? I'm confused

**Comment by Explosion-Scratch** [comment]
Like the section on the side titled 'about'. Oh well.

**Comment by GrahamSH-LLK** [comment]
Just put what you want it to say here and tag @WorldLanguages.

**Comment by Explosion-Scratch** [comment]
I just added the website link and some tags to it.

**Comment by cobaltt7** [comment]
Sorry I didn't know that was a rule here

Paul Reid

________________________________
